charts/kubeclarity/templates/deployment.yaml

```diff
@@ -1,3 +1,21 @@
+{{- if index .Values "kubeclarity-postgresql" "enabled" }}
+{{- if index .Values "kubeclarity-postgresql-external" "enabled" }}
+{{- fail "kubeclarity-postgresql and kubeclarity-postgresql-external can not both be enabled!" }}
+{{- end }}
+{{- end }}
+{{- $secretKey := index .Values "kubeclarity-postgresql-secret" "secretKey" -}}
+{{- $secretName := index .Values "kubeclarity-postgresql" "auth" "existingSecret" -}}
+{{- $dbHost := print (include "kubeclarity.name" .) "-postgresql" -}}
+{{- $dbPort := index .Values "kubeclarity-postgresql" "service" "ports" "postgresql" | quote -}}
+{{- $dbUser := index .Values "kubeclarity-postgresql" "auth" "username" -}}
```

Comment:
nit: in the previous code port was not quoted but username and dbHost was quoted

```
-p 5432 -U "postgres"

"{{ include "kubeclarity.name" . }}-postgresql"
```

Reply:
@FrimIdan Updated

Reply:
Sorry for the nit comments but I really wanted to keep the output the same as before the review in the case of a local postgresql.
I've run `helm template` command to compare it and got
```
286c286
<           command: ['sh', '-c', 'until pg_isready -h kubeclarity-kubeclarity-postgresql -p 5432 -U "postgres" -d "dbname=kubeclarity";
---
>           command: ['sh', '-c', 'until pg_isready -h kubeclarity-kubeclarity-postgresql -p 5432 -U postgres -d "dbname=kubeclarity";
371c371
<               value: "kubeclarity-kubeclarity-postgresql"
---
>               value: kubeclarity-kubeclarity-postgresql
375c375
<               value: "postgres"
---
>               value: postgres
443c443
```

I think that the following will solve it
```
10c10
< {{- $dbUser := index .Values "kubeclarity-postgresql" "auth" "username" -}}
---
> {{- $dbUser := index .Values "kubeclarity-postgresql" "auth" "username" | quote  -}}
133c133
<               value: {{ $dbHost }}
---
>               value: {{ $dbHost | quote  }}
```

Reply:
@FrimIdan I added quotes in the command to fix the current version.